docs/links.rst

```diff
@@ -21,6 +21,14 @@ Link implementations
 
 .. autoclass:: chainerrl.links.Sequence
 
+.. autoclass:: chainerrl.links.StatelessRecurrent
```

Comment:
The class looks an interface rather than an implementation.

Reply:
fixed